src/constants/featureFlags.ts

```diff
@@ -1,6 +1,10 @@
 export const FEATURE_FLAGS = {
-  HOMEPAGE_TEMPLATES: "homepage_new_templates",
   REPO_PRIVATISATION: "repo_privatisation",
   BANNER: "banner",
   NPS_FORM: "nps_form",
+  ENABLED_BLOCKS: "homepage_enabled_blocks",
```

Comment:
Hey may I understand why splitting this up into 
`homepage_enabled_blocks` and non `homepage_enabled_blocks` would be non-trivial work? Feels like a quick win here when things are buggy!

Reply:
i only ever want 1 check to be in the codebase - my thought process here is if we have to rollback, we'll rollback by block. this is because it's q likely that, for example, an issue on the block for 1 site would also be similarly applicable to other sites (ie, that blocking by site doesn't add much value).

because of this, i would rather simplify the checks required rather than adding complexity here. wdyt on this!

Reply:
non blocking: hmm i think complexity introduced in storing + merging from local storage > maintaining a model of feature per site, but I dont feel too strong about this we can go ahead with this implementation